### PR TITLE
AGP-148: AN ≠ GoT

### PR DESCRIPTION
**Pull request by burrrata**
Positive sum > zero sum

### CODE REVIEW

File: AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+This incentivizes teams to ask for as much money as possible as early as possible. It's a SaaS model, but for Aragon treasury funding. It's easy to sign up for (ANT holders vote yes on almost everything), and hard to cancel (to date an AGP has never been reversed).
+
+AGP-103 favors current recurring payments over new proposals asking for funds. The teams who are currently in the Aragon ecosystem will benefit greatly from this. New teams will be discouraged to get involved in the network. If they do want to get involved they will have to find a way to make room in the budget for their projects. 
+
+To address the fact that AGPs have never been reversed new proposals cancelling currently recurring proposals will be favored before all others. Note: this does not mean that a team has to cancel their own proposal. This means that any proposal that cancels another proposal will be processed before all others. Anyone can submit an AGP. This means that every quarter every team (current or new) will be incentivized to submit an AGP cancelling the funding of other teams. They can do this transparently or anonymously.
```
Comment (john-light) [inline AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md]: The fact that an AGP has been reversed yet isn't really a good argument for anything. The AGP process is barely a year old, it's not unusual that authors and voters haven't found an AGP worthy of completely undoing yet. However, AGPs have been approved that _modify_ previous AGPs (all the approved Meta track AGPs for example), showing that voters are willing to introspect and make changes when needed.

File: AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+To address the fact that AGPs have never been reversed new proposals cancelling currently recurring proposals will be favored before all others. Note: this does not mean that a team has to cancel their own proposal. This means that any proposal that cancels another proposal will be processed before all others. Anyone can submit an AGP. This means that every quarter every team (current or new) will be incentivized to submit an AGP cancelling the funding of other teams. They can do this transparently or anonymously. 
+
+On top of all that, AGP-103 favors proposals with the most absolute support. Currently the majority of active ANT voters are very centralized. A few people and teams have more weight than anyone else. This is far from decentralized. This is not the utopian future Aragon promised. The Aragon Network needs to be a fair and open place for people to collaborate in positive sum games. AGP-103 creates a zero sum game. This is not [what Aragon is supposed to be about.](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md)
```
Comment (john-light) [inline AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md]:
```suggestion
This is not the utopian future Aragon promised.
```

Has "Aragon" ever promised utopia (a word whose roots literally mean "no place")? Have governance decisions ever been anything other than tokenholder-driven?

> AGP-103 creates a zero sum game.

I don't think it does. Nothing in AGP-103 says that authors who wish to spend outside of the budget cannot propose an AGP that increases the budget so they can spend more. That said, there is a limited pool of funds to draw from. The AA/ AN does not have infinite money. And so in that larger sense, there is a zero-sum game, but AGP-103 did not create that, that is just reality.

**Comment by john-light** [comment]
Left a couple of non-editorial comments inline on the AGP text.

**Comment by burrrata** [comment]
@john-light To confirm, those comments are not related to this AGP moving to the community review stage, correct?

File: AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md (hunk 1 above)
Comment (burrrata) [reply AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md]: Good call. Updated accordingly.

**Comment by john-light** [comment]
@burrrata did not see your last question before. that is correct, they were non-editorial comments.

**Comment by luisivan** [comment]
Withdrawing this AGP from ANV-5 on behalf of the Aragon Association

**Comment by john-light** [comment]
With the approval of [AGP-155](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-155.md) in Aragon Network Vote #6, the AGP process has now been sunset. All open AGP pull requests are now being closed.

Thank you for participating in the AGP process!

**Pull request closed without merging**